Sources/OktaIdx/IDXRemediation.swift

```diff
@@ -90,6 +90,21 @@ extension IDXClient {
             super.init()
         }
         
+        public override var description: String {
+            let logger = DebugDescription(self)
+            let components = [
+                logger.address(),
+                "\(#keyPath(type)): \(type)",
+                "\(#keyPath(name)): \(name)"
+            ]
+
+            return logger.brace(components.joined(separator: "; "))
+        }
+        
+        public override var debugDescription: String {
+            description
```

Comment:
We should print more informationn he Remediation object , since that's one of the most important things to a developer.  For example, we should show the form fields, options, and authenticators in the form.

Plus, it looks like the indentation is a bit off for these nested values.

```
<Form: 0x0000600003b611c0> {
    <Field: 0x0000600000a810e0; name: userProfile; label: -; type: -; value: -> {
    isVisible: true;
isMutable: true;
isRequired: false;
isSecret: false;
options: -;
messages: 
}
}
```

Reply:
Please, review.